recipes/concoct/meta.yaml

```diff
@@ -34,7 +34,7 @@ requirements:
     - pandas >=0.11.0
     - biopython >=1.62b
     - pytz >=2013.9
-    - scikit-learn >=1.1.3
+    - scikit-learn
```

Comment:
According to  the [logs](https://dev.azure.com/bioconda/25074ded-3133-43ca-af80-895ab87b53cb/_apis/build/builds/61855/logs/21) now it uses `scikit-learn:     1.5.0-py312h1fcc3ea_1`

Reply:
Ok cool, I'm on baby duty today but when I get a chance I'll try and get the docker image/build the environment and try running concoct to see if the error people are getting is there or not.